src/dataprep/main/prep_nsf/scinet_data_to_db.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# %%
+"""
+Download SciSciNet table SciSciNet_Link_NSF with links between NSF-grants and papers
+Upload into db
+link to Paper_Author_Affiliations, Authors, NSF_Investigator in R file: test_sciscinet_data.R in same folder
+
+
+Create table in database:
+- SciSciNet_Link_NSF
+
+SciSciNet_Link_NSF schema is:
+
+GrantID INTEGER, PaperID INTEGER
+ 
+
+unique index on Grantid and PaperID (multiple paperIDs per GrantID)
+"""
+
+import subprocess
+import sqlite3 as sqlite
+import argparse
+import os
+from os import listdir
+from os.path import isfile, join
+import pandas as pd
+import numpy as np 
+import re 
+import sys
+import requests
+
+sys.path.append('/home/mona/mag_sample/src/dataprep/')  
```

Comment:
This should not be necessary (and not be in the script). If you have problem when not using this, we can solve it together.

Reply:
Unfortunately, I cannot make it run when leaving it out, I'd need your help here please

Reply:
Here are some instructions: https://github.com/f-hafner/mag_sample/blob/main/README.dev.md 
Let me know if it works.

Reply:
It works, thank you!